### DISCUSSION TITLE
Greetings

### DISCUSSION THREAD

**Discussion by j3b4** [evennia/evennia#3618] [General Discussion]
((Editing this whole thing cause I asked a boring question. ))

I'm coming back to Evennia because I've got the bug again to work on an old pet project.  However I'm now thinking that the project should be a visually rich  graphical 2D or 3D game.  Is it possible that Evennia might still be an appropriate engine for developing such a thing?  How much of a stretch would it be? Thoughts from anyone?

**Comment by CloudKeeper** [comment]
Hey JB! Good to see you mate. 

Evennia has made some advancements since last time you were here in that area. It supports all sorts of multimedia, but it's still a MU* engine at the end of the day. I've seen the community build graphical MUDs (rooms/characters use a picture instead of a written description), less free form games ( hardcoded buttons instead of text), and ASCII style rogue likes (where the ASCII graphics could be replaced with graphical tiles). There's also been proof of concepts for clients made with unity and unreal which could handle more graphics based on information sent from an Evennia backend.

Lots of flexibility, but the further you move from a MU* the more another engine might make more sense.

P.s. no IRC anymore. We're all on discord.

**Reply by j3b4** [reply]
Cloud! So fun to see you again. I wonder if my strongest motivation for shoehorning my project into Evennia, isn't just the fact that I like the community here so much 😁.

I'm about to go off on an adventure learning about graphical game design on Godot... But I can see myself coming back afterwards and integrating. My dream game would have a very robust "lobby" that not only serves to help with matchmaking but also contains elements of meta game play. Maybe that lobby should be built in Evennia. I don't know.

**Reply by CloudKeeper** [reply]
I've been tinkering with Godot as well. I want to make a 2d top down version of Mount and Blade. Look forward to see what you come up with.
